scripts/copy_componentType_custom_properties/README.md

```diff
@@ -0,0 +1,24 @@
+# Copy Custom Properties from one Component Type to another Component Type
+
+This script is designed to copy custom properties from one Component Type to another Component Type in your OpsLevel account.
+
+Prequisites:
+
+- The property definition must be defined on a component type to copy from.
+
+Requirements:
+
+- Python 3.10.10 or higher
+- `requests` library is installed
```

Comment:
instead of `requests`, we could consider using a python graphql library: https://github.com/graphql-python/gql. My be tidier.